multisig/src/setup.rs

```diff
@@ -1,7 +1,12 @@
 use multiversx_sc::imports::*;
 
 use eth_address::EthAddress;
-use sc_proxies::{bridge_proxy_contract_proxy, esdt_safe_proxy, multi_transfer_esdt_proxy};
+use sc_proxies::{
+    bridge_proxy_contract_proxy, bridged_tokens_wrapper_proxy, esdt_safe_proxy,
+    multi_transfer_esdt_proxy,
+};
+
+const MAX_BOARD_MEMBERS: usize = 40;
```

Comment:
why 40? I would go to a higher value like 100 or something

Reply:
This is a fix for issue 4 from the [Audit document](https://docs.google.com/document/d/1y4lq-MXy9zvgz4KADmlEGaSDXfIzAHwE6y94PtOvQOE/edit?tab=t.0).
We iterate over all board signatures and the call may run out of gas.
Maybe an integration test from your side could help us adjust this value.